edgexfoundry/edgex-examples
# ci(workflows): Add Tar Zipped Asset for Helm Chart Sample Consumption

This provides the ability to consume the helm chart example as an asset. It is for use with automation in another system that requires this file format for consumption and installation of the chart.

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) - I did not due to time constraints.

## Testing Instructions
To see the GitHub Action in..."action" (I'll see myself out), create a release

* Successful run of the action can be found here: https://github.com/eshenayo/edgex-examples/runs/5862759176?check_suite_focus=true
* Creates a tar zipped file containing the Helm deployment directory: https://github.com/eshenayo/edgex-examples/releases/tag/v0.1.0-alpha
![image](https://user-images.githubusercontent.com/8902109/171455968-9a1aa256-a9a8-4dd3-8e03-1dffacc3821b.png)
